src/utils/getReactNativeVersion.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ *
+ * @flow
+ */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function getReactNativeVersion(cwd: string) {
```

Comment:
Would not it be better to go into `node_modules/react-native` and read `package.json` from there?

Reply:
Based this off https://github.com/callstack/haul/blob/master/src/utils/isReactNativeProject.js which I assumed was your preferred way of reading package.json to get details of packages.

Reply:
`isReactNativeProject` only tells us if project is RN. We need exact version of RN used, so I'd say I prefer to go into node_modules

Reply:
Will change :)

Reply:
Wouldn't be better if /src/utils/isReactNativeProject.js@master would return `false | string` to get rid of `getReactNativeVersion` completely? @zamotany @Krizzu

Reply:
I change `getReactNativeVersion` to return `string | null` and got rid of `isReactNativeProject`.

Reply:
@zamotany that's even better!

Reply:
Yeah, good approach @zamotany